cmd/nginx-ingress/flags.go

```diff
@@ -220,77 +220,12 @@ func parseFlags() {
 	if *versionFlag {
```

Comment:
should this `versionFlag` be in initValidation?
also do we still need the `parseFlags` function in addition to `mustValidateFlags`?